frontend/src/patientApp/timeOfTest/TermsOfService.tsx

```diff
@@ -38,13 +41,16 @@ const TermsOfService: React.FunctionComponent<Props> = ({
       )}
     >
       <form className="grid-container maxw-tablet usa-prose">
-        <h1 className="font-heading-lg margin-top-3">Terms of service</h1>
-        <p className="margin-top-105">
-          This testing site uses{" "}
-          <a href="https://simplereport.gov/">SimpleReport</a> to manage
-          COVID-19 testing and reporting. The terms below explain SimpleReport’s
-          policies and terms of service.
-        </p>
+        <h1 className="font-heading-lg margin-top-3">{t("terms.label")}</h1>
+        <Trans
+          t={t}
+          parent="p"
+          className="margin-top-105"
+          i18nKey="terms.explanation"
+          components={[
+            <a href="https://simplereport.gov/">SimpleReport Link</a>,
+          ]}
+        />
```

Comment:
An example of a more complex translation that requires some component interpolation